app/views/registration_wizard/aso_previously_funded.html.erb

```diff
@@ -7,7 +7,7 @@
 %>
   <h1 class="govuk-heading-xl">DfE scholarship funding is not available</h1>
 
-  <p class="govuk-body">Our records show that you are already registered to study <b>Early headship coaching offer</b> (previously known as the Additional Support Offer).</p>
+  <p class="govuk-body">Our records show that you are already registered to study <%= localise_sentence_embedded_course_name(Course.ehco.first) %> (previously known as <%= localise_sentence_embedded_course_name(Course.aso.first) %>).</p>
```

Comment:
hmm i was hoping the remove the `Course.aso` in my Pr but this might stop that. Just as its deprecated wanted to remove all references to the DB unless maybe we had a comment above the method.

Reply:
might be a point where we hard code it back once removed since it's no longer a course and writing it directly into the template is fine?